ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -4093,8 +4093,11 @@ let vm_clone_aux clone_op cloned_string printer include_template_vms rpc
 
 let vm_clone printer = vm_clone_aux Client.VM.clone "Cloned " printer true
 
-let vm_snapshot printer =
-  vm_clone_aux Client.VM.snapshot "Snapshotted " printer false
+let vm_snapshot printer params =
+  let ignore_vdis =
+    try (List.assoc "ignore-vdis" params) with _ -> []
+  in
+  vm_clone_aux (Client.VM.snapshot ~ignore_vdis) "Snapshotted " printer false
```

Comment:
I am not sure how to have acces to `params` in `vm_snapshot`, i only have access to `printer`. `params` is only reachable in `vm_clone_aux`...
Can someone help me? 
It's forwarded by
```ocaml
  ; ( "vm-snapshot"
    , {
        reqd= ["new-name-label"]
      ; optn= ["new-name-description"]
      ; optn= ["new-name-description"; "ignore-vdis"]
      ; help=
          "Snapshot an existing VM, using storage-level fast disk snapshot \
           operation where available."
      ; implementation= No_fd Cli_operations.vm_snapshot
      ; flags= [Standard; Vm_selectors]
      }
    )
```
In the cli_frontend.